src/lib/state/reducers/communities.js

```diff
@@ -0,0 +1,40 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2022 CERN.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import {
+  FETCH_COMMUNITY_ERROR,
+  FETCH_COMMUNITY_LOADING,
+  FETCH_COMMUNITY_SUCCESS,
+  SET_COMMUNITY,
+} from '../types';
+
+export default (state = {}, action) => {
```

Comment:
please name the reducer, it will be much easier to search for it in the code

Reply:
Good point! Will rename also the others :+1: